python/lsst/obs/lsstSim/lsstSimMapper.py

```diff
@@ -134,7 +135,7 @@ def validate(self, dataId):
             if component not in dataId:
                 continue
             val = dataId[component]
-            if not isinstance(val, str):
+            if not isinstance(val, basestring):
```

Comment:
I don't think this is py3-compatible: https://docs.python.org/3.0/whatsnew/3.0.html#text-vs-data-instead-of-unicode-vs-8-bit